src/main/java/com/mparticle/kits/AppboyKit.java

```diff
@@ -275,6 +275,8 @@ public void setUserAttribute(String key, String value) {
             } catch (Exception e) {
                 Logger.error("unable to set DateOfBirth for \"dob\" = " + value + ". Exception: " + e.getMessage());
             }
+        } else if(UserAttributes.ZIPCODE.equals(key)){
```

Comment:
Nitpick, but there should be a space after `if` and before the `(` and a space after the last `)` and before the `{`:

`} else if (UserAttributes.ZIPCODE.equals(key)) {`